DEVELOPING.md

```diff
@@ -80,10 +82,14 @@ docker-compose build && docker-compose up
 
 2. Visit the app at http://localhost:80 (or http://localhost)
 
-## Migrations
```

Comment:
This should be kept?

Reply:
i will re-add it as a sub-heading under Database Changes